### PR TITLE
remove update dd device from execution

### PR DESCRIPTION
**Pull request by acosferreira**
# Description

Remove wipe execution

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [X] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I run `go fmt ./...` to check that my code is properly formatted
- [ ] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

**Review by ldjebran — APPROVED** [review]
LGMT

**Review by thom-at-redhat — APPROVED** [review]

File: test-container/Dockerfile [hunk 1]
```diff
@@ -9,7 +9,8 @@ RUN git clone https://github.com/fedora-iot/fido-device-onboard-rs.git
 RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
 RUN source $HOME/.cargo/env && \
     rustup default nightly && rustup update && \
-    cd fido-device-onboard-rs && cargo build --lib
+    cd fido-device-onboard-rs && cargo build --release
+    #cd fido-device-onboard-rs && cargo build --lib
```
Comment (Avielyo10) [inline test-container/Dockerfile:13]: why?

**Review by Avielyo10 — CHANGES_REQUESTED** [review]

**Comment by Avielyo10** [comment]
@mergify rebase

File: test-container/Dockerfile (hunk 1 above)
Comment (acosferreira) [reply test-container/Dockerfile]: was a try to fix the ci problem ;)

**Review by Avielyo10 — APPROVED** [review]

**Pull request merged**